.git-blame-ignore-revs

```diff
@@ -3,4 +3,4 @@
 # github does this automatically.
 
 # setup prettier
-273e57eb2adb571
+273e57eb2adb5710dddcd2737e566fa1c606b62b
```

Comment:
previous version broke git blame, apparently we have to give the commit id in full